src/script/lua_api/l_util.cpp

```diff
@@ -159,6 +159,19 @@ int ModApiUtil::l_write_json(lua_State *L)
 	return 1;
 }
 
+// get_tool_wear_after_use(uses[, initial_wear])
+int ModApiUtil::l_get_tool_wear_after_use(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	u32 uses = readParam<int>(L, 1);
+	u16 initial_wear = 0;
+	if (!lua_isnoneornil(L, 2))
+		initial_wear = readParam<int>(L, 2);
```

Comment:
I believe this works too
```suggestion
	u16 initial_wear = readParam<int>(L, 2, 0);
```